pinot-core/src/main/java/org/apache/pinot/core/operator/docidsets/BitmapDocIdSet.java

```diff
@@ -33,7 +36,8 @@
   public BitmapDocIdSet(ImmutableRoaringBitmap[] bitmaps, int startDocId, int endDocId, boolean exclusive) {
     int numBitmaps = bitmaps.length;
     if (numBitmaps > 1) {
-      MutableRoaringBitmap orBitmap = MutableRoaringBitmap.or(bitmaps);
+      Iterator iterator = Arrays.asList(bitmaps).iterator();
+      MutableRoaringBitmap orBitmap = MutableRoaringBitmap.or(iterator, startDocId, endDocId + 1);
```

Comment:
@kishoreg : can you please clarify ? I'm not sure we need this change for presence vector.

Reply:
I believe this might be slightly more efficient (based on conversation with Kishore)

Reply:
This could backfire because there is an extra step of selecting range for each bitmap. I checked the code and seems this step is always redundant because we always use 0 as startDocId and numDocs-1 as endDocId. Also this is not related to this PR. @kishoreg Do you see actual performance gain for this change?